alcs/acs2/Classifier.py

```diff
@@ -379,15 +393,32 @@ def is_marked(self):
 
         return True
 
-    def crossover(self, cl2):
+    def crossover(self, cl2, samplefunc=sample):
         """
         Executes crossover with other classifier
+        :param samplefunc:
         :param cl2: other classifier
         """
-        self.condition.two_point_crossover(cl2.condition)
+        self.condition.two_point_crossover(
+            cl2.condition, samplefunc=samplefunc)
 
         q = float(sum([self.q, cl2.q]) / 2)
         r = float(sum([self.r, cl2.r]) / 2)
 
         cl2.q = q
         cl2.r = r
+
+    def __eq__(self, other):
```

Comment:
It it necessary?

Reply:
Yes, `ClassifiersList` comparisons in tests rely on this.